fenix/app/src/main/java/org/mozilla/fenix/library/historymetadata/HistoryMetadataGroupFragment.kt

```diff
@@ -188,10 +187,7 @@ class HistoryMetadataGroupFragment :
                     selectedItem.url
                 }
 
-                (activity as HomeActivity).apply {
-                    browsingModeManager.mode = BrowsingMode.Private
-                    supportActionBar?.hide()
-                }
+                (activity as HomeActivity).supportActionBar?.hide()
```

Comment:
Note that `openItemsInNewTab` which is called on the line before this dispatches the PBM state update, so I removed the redundancy here.